### ISSUE TITLE
fasta2select: switch away from clustalw to clustal omega or muscle

### ISSUE THREAD

**Issue by IAlibay** [MDAnalysis/mdanalysis#3459]
## Is your feature request related to a problem? ##

fasta2select currently relies on clustalw(2), which is quite old and has been retired in favour of muscle and clustal omega (clustalO).

## Describe the solution you'd like ##

It would be good to look at implementing options to switch away from clustalw in the long term to use either of these newer options. Our current programmatic approach to alignment (via Biopython) provides classes for muscle and clustalO which should allow them to be easily added.

## Describe alternatives you've considered ##

We could also just remove fasta2select alignment from the core library - although I have a feeling this is a reasonably well used method.

**Comment by orbeckst** [comment]
This is just a pairwise alignment so I think we could just use any old pairwise aligner and don't need muscle or clustal* for it.  I know that I wrote that hack when I only knew about clustal. It might be time to rewrite it in a cleaner and leaner fashion and in the process ditch the annoying optional dependency on clustal.
 
Does http://scikit-bio.org/ have a native aligner? Could be used in favor of biopython — a long time ago we discussed using it but weren't able to because of Py 2.7. Now that's not an issue anymore.
